src/sisl/geom/_neighbors/_neighborlists.py

```diff
@@ -344,10 +344,10 @@ class AtomNeighborList(Neighbors):
         The lists that, when iterated, return `AtomNeighborList` objects.
     """
 
-    #: The atom for which the neighbors are stored.
     atom: int
+    """The atom for which the neighbors are stored."""
```

Comment:
Why do you want it like this? It looks worse to me

Reply:
Agreed, but when building the docs, I get errors on missing attribute docs.... This is the PEP suggested way of adding docs-strings to attributes.

Reply:
Hmm this is very strange, last time I checked sphinx took the documentation from the previous line's comment if it started with `#:`. This is no longer true?

Reply:
Perhaps `#:` only works for module attributes (constants), and not class attributes. It isn't fully clear to me either, because in `constant.py` I do `#:`, and I don't get errors there!